recipes/iqtree/meta.yaml

```diff
@@ -1,40 +1,43 @@
 {% set name = "IQTREE" %}
 {% set version = "2.3.5" %}
-{% set sha256 = "8e323e0b7c46e97901d3500f11e810703e0e5d25848188047eca9602d03fa6b1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version | replace("-", "_") }}
 
 build:
-  number: 1
+  number: 2
   run_exports:
     - {{ pin_subpackage('iqtree', max_pin="x") }}
 
 source:
-  - url: https://github.com/iqtree/iqtree2/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: {{ sha256 }}
-  - url: https://github.com/tothuhien/lsd2/archive/refs/tags/v.2.4.1.tar.gz
-    sha256: 3d0921c96edb8f30498dc8a27878a76d785516043fbede4a72eefd84b5955458
-    folder: lsd2
+  git_url: https://github.com/iqtree/iqtree2.git
```

Comment:
```suggestion
  # using git_url rather than tarball due to use of 2 submodules by iqtree (lsd2, cmaple)
  # Previously, the submodules were specified as source tarballs themselves
  # But the submodules weren't updated when iqtree was, hence switch to git_url
  git_url: https://github.com/iqtree/iqtree2.git
```